CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+## Contributing
```

Comment:
Can you adjust the headers? We had formatted this way because it was inside another file, thus making it just another section, but now, having a separate file of its own, headers should be properly adjusted. Converting `##` to `#` and `###` to will do.